ckanext/stadtzhtheme/templates/snippets/search_form_with_autosuggestion.html

```diff
@@ -0,0 +1,101 @@
+{# this has been copied from the ckan search_form:
```

Comment:
can you double-check if it is possible to `{% ckan_extends %}` this template, so that you can just override `{% block search_input %}` ?
at least ckanext-hierarchy does something similar in `ckanext/ckanext-hierarchy/ckanext/hierarchy/templates/snippets/search_form.html` so I would assume that this is possible.